nion/swift/LinePlotCanvasItem.py

```diff
@@ -420,8 +420,10 @@ def convert_to_calibrated_size_str(f: float) -> str:
 
     def __view_to_intervals(self, data_and_metadata: DataAndMetadata.DataAndMetadata, intervals: typing.List[typing.Tuple[float, float]]) -> None:
         """Change the view to encompass the channels and data represented by the given intervals."""
+        # X dimensional factors are controlled by dimensional_calibrations
+        # Y dimensional factors are controlled by intensity_calibration
 
-        if len(intervals) > 0:
+        if len(intervals) > 0:  # left and right are absolute positions on the display
```

Comment:
Likewise with this comment - I don't understand what it means. Can you either clarify or remove it? The intervals are relative coordinates, typically ranging from 0.0 to 1.0; they're not absolute.

What would be helpful, though, is a comment before this section describing that this section of code is finding the min/max values of the set of intervals.